dist/src/main/config/broker.yaml.template

```diff
@@ -160,6 +160,18 @@
       # This setting can also be overridden using the environment variable ZEEBE_BROKER_DATA_DISKUSAGEMONITORINGINTERVAL
       # diskUsageMonitoringInterval = 1s
 
+      # database:
+        # Specify custom column family options overwriting Zeebe's own defaults.
+        # WARNING: This setting requires in-depth knowledge of Zeebe's embedded database!
+        # The expected property key names and values are derived from RocksDB's C implementation,
+        # and are not limited to the provided examples below. Please look in RocksDB's SCM repo
+        # for the files: `cf_options.h` and `options_helper.cc`. This setting can also be overridden
+        # using the environment variable ZEEBE_BROKER_DATA_DATABASE_COLUMNFAMILYOPTIONS.
+        # However, it does not seem possible to provide multiple column family options that way.
```

Comment:
Setting the environment variable does not follow the normal pattern here,  right? That is ZEEBE_BROKER_DATA_DATABASE_COLUMNFAMILYOPTIONS_WRITE_BUFFER_SIZE does not work here. So should we just say that don't use environment variable? Otherwise we should document how to use env variable - eg:- ZEEBE_BROKER_DATA_DATABASE_COLUMNFAMILYOPTIONS="write_buffer_size=4" .  I would prefer just saying please don't use env variable. Wdyt?

Reply:
I've done some further testing and found that:
```
export ZEEBE_BROKER_DATA_DATABASE_COLUMNFAMILYOPTIONS_COMPACTION_PRI="kOldestLargestSeqFirst"
export ZEEBE_BROKER_DATA_DATABASE_COLUMNFAMILYOPTIONS_WRITE_BUFFER_SIZE=8388608
``` 

results in:

```
    "database" : {
      "columnFamilyOptions" : {
        "compaction.pri" : "kOldestLargestSeqFirst",
        "write.buffer.size" : "8388608"
      }
    },
```

So the key's have dots instead of underscores in their name.

We could consider implementing a character replacement for the `.` (dot) char to `_` (underscore) in the `getColumnFamilyOptions()` method of the `DatabaseCfg` class. That would make it possible to define the options through env vars, although specifically setting `ZEEBE_BROKER_DATA_DATABASE_COLUMNFAMILYOPTIONS="x=y"` also allows to set 1 option (but not more). If that seems to much complexity we can just comment not to use env vars.

What do you think?

Reply:
That's a good idea. But I wouldn't insist on doing it. If we want we can do it in a separate PR. I suggest that you create an issue for it, and we can discuss it during triage.

Reply:
In that case, I'll add a comment now to not use env vars and create an issue to discuss a fix.

Reply:
:+1: